test/BaselineOutput/Common/FeatureContribution/FastForestBinary.tsv

```diff
@@ -4,14 +4,15 @@
 #@   col=X2VBuffer:R4:1-4
 #@   col=X3Important:R4:5
 #@   col=Label:R4:6
-#@   col=Features:R4:7-12
-#@   col=Features:R4:13-18
-#@   col=FeatureContributions:R4:19-24
-#@   col=FeatureContributions:R4:25-30
-#@   col=FeatureContributions:R4:31-36
-#@   col=FeatureContributions:R4:37-42
+#@   col=Label:BL:7
```

Comment:
Where does it come from? New trainer can also produce new labels?

Reply:
We restrict binary classification to work only with boolean columns.
It's easier to add convert than change shared code which generate R4 labels for different classes.

---
In reply to: [263181449](https://github.com/dotnet/machinelearning/pull/2804#discussion_r263181449) [](ancestors = 263181449)